src/main/java/com/github/nicholasmoser/gnt4/seq/ext/SeqAssembler.java

```diff
@@ -0,0 +1,1171 @@
+                    try {
+                        currentOpcode = new BranchLessThanEqualZeroLink(offset, Integer.decode(operands));
+                    } catch (Exception e) {
+                        currentOpcode = new BranchLessThanEqualZeroLink(offset, operands);
+                    }
+                    break;
+                case "blr":
+                    currentOpcode = new BranchLinkReturn(offset);
+                    break;
+                case "blreqz":
+                    currentOpcode = new BranchLinkReturnEqualZero(offset);
+                    break;
+                case "blrnez":
+                    currentOpcode = new BranchLinkReturnNotEqualZero(offset);
+                    break;
+                case "blrgtz":
+                    currentOpcode = new BranchLinkReturnGreaterThanZero(offset);
+                    break;
+                case "blrgez":
+                    currentOpcode = new BranchLinkReturnGreaterThanEqualZero(offset);
+                    break;
+                case "blrltz":
+                    currentOpcode = new BranchLinkReturnLessThanZero(offset);
+                    break;
+                case "blrlez":
+                    currentOpcode = new BranchLinkReturnLessThanEqualZero(offset);
+                    break;
+                case "branch":
+                    switch (opcode[1]) {
+                        case "table":
+                            if (opcode.length == 2) {
+                                baos.write(new byte[]{0x01, 0x50});
+                                baos.write(SEQ_RegCMD1(op[0]));
+                                baos.write(ByteUtils.fromInt32(op.length - 1));
+                                List<Integer> offsets = new LinkedList<>();
+                                List<String> branches = new LinkedList<>();
+                                for (int j = 1; j < op.length; j++) {
+                                    try {
+                                        int off = Integer.decode(op[j]);
+                                        offsets.add(off);
+                                        baos.write(ByteUtils.fromInt32(off));
+                                    } catch (Exception e) {
+                                        baos.write(ByteUtils.fromInt32(0));
+                                        branches.add(op[j]);
+                                    }
+                                }
+                                if (offsets.size() > 0) {
+                                    currentOpcode = new BranchTable(offset, baos.toByteArray(), op[0], offsets);
+                                } else if (branches.size() > 0) {
+                                    currentOpcode = new BranchTable(offset, baos.toByteArray(), op[0], offsets, branches);
+                                } else {
+                                    System.err.printf("Neither offsets nor label names given for branch table at offset %d%n", offset);
```

Comment:
Can you replace usages of `System.err.printf` with:

```java
LOGGER.log(Level.SEVERE, String.format("Neither offsets nor label names given for branch table at offset %d%n", offset));
```

Reply:
Fixed